specification/columns/commitmentutilizationused.md

```diff
@@ -0,0 +1,48 @@
+# Commitment Utilization Used
+
+Commitment Utilization Used is a numeric value representing the unit amount of a purchased [*commitment-based discount*](#glossary:commitment-based-discount) applied or not applied to a resource during a billing period.  The ratio of CommitmentUtilizationUsed over [*CommitmentUtilizationTotal*](#commitment-utilization-total) produces the utilization rate for a given commitment-based discount or aggregate of.
```

Comment:
Should this be:

"the unit amount of a purchased [*commitment-based discount*](#glossary:commitment-based-discount) applied to a resource during a billing period."

without this part "or not applied "

because if it's not applied then it wasn't used?

Or is something else meant be applied?

Reply:
I suppose you can still get the utilization if you write a WHERE clause to remove the unused commitment line items

Reply:
No, you're right, that last clause should be removed.